bin/phase-helpers.sh

```diff
@@ -573,13 +573,13 @@ econf() {
 		if [[ -n $CONFIG_SHELL && \
 			"$(head -n1 "$ECONF_SOURCE/configure")" =~ ^'#!'[[:space:]]*/bin/sh([[:space:]]|$) ]] ; then
 			# preserve timestamp, see bug #440304
-			touch -r "${ECONF_SOURCE}/configure" "${ECONF_SOURCE}/configure._portage_tmp_.${pid}" || die
+			cp "${ECONF_SOURCE}/configure" "${ECONF_SOURCE}/configure._portage_tmp_.${pid}" || die
```

Comment:
Huh? Have never seen cp dropping mode bits that are possible to be preserved.
And indeed, man cp(1) tells for `--preserve[=ATTR_LIST]`:
`preserve the specified attributes (default: mode,ownership,timestamps), ...`